eng/pipelines/template-tests.yml

```diff
@@ -1,3 +1,9 @@
+pr:
+  paths:
+    include: 
+      - eng/templates/**/*template-test*.yml
+      - eng/pipelines/template-tests.yml
```

Comment:
Will this cause the pipeline to only change on PR's that include this change or will the scheduled run also continue to run as planned?

Reply:
This will not affect scheduled runs. Scheduled runs are configured manually in the Azure DevOps UI.